dup2 failed on arm64

```
# github.com/bugsnag/panicwrap
/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/bugsnag/panicwrap@v1.3.1/monitor.go:55:8: undefined: dup2
```

Consider replacing `dup2` for arm64 build

relates to https://github.com/Homebrew/homebrew-core/pull/70684

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,10 @@
+## TBD
+
+### Bug fixes
+
+* Add build support from arm64-based macOS by updating build settings for
+  `dup2()`
+
 ## 1.3.1 (2021-01-12)
 
 This release removes the dependency on [osext](https://github.com/kardianos/osext)

--- a/dup2.go
+++ b/dup2.go
@@ -1,4 +1,4 @@
-// +build !arm64,!windows
+// +build !windows
 
 package panicwrap
 

